.travis.yml

```diff
@@ -45,11 +45,13 @@ env:
   - PACKAGE='CHECK' 
 install: 
 before_script: 
+- sudo apt-get install zsh -y
```

Comment:
No. Add `zsh` in the list `addons.apt.packages` instead. That avoids that apt is called twice.